sdk/maps/azure-maps-render/azure/maps/render/_render_client.py

```diff
@@ -0,0 +1,402 @@
+    def get_map_state_tile(
+        self,
+        stateset_id: str,
+        tile_index_z: int,
+        tile_index_x: int,
+        tile_index_y: int,
+        **kwargs: Any
+    ) -> Iterator[bytes]:
+        """Fetches state tiles in vector format typically to be integrated into indoor maps module of map
+        control or SDK.
+
+        :param tile_index_z:
+            Zoom level for the desired tile.
+        :type tile_index_z: int
+        :param tile_index_x:
+            X coordinate of the tile on zoom grid.
+        :type tile_index_x: int
+        :param tile_index_y:
+            Y coordinate of the tile on zoom grid.
+        :type tile_index_y: int
+        :param stateset_id:
+            The stateset id.
+        :type stateset_id: str
+        :return: Iterator of the response bytes
+        :rtype: Iterator[bytes]
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+
+        return self._render_client.get_map_state_tile(
+            stateset_id=stateset_id,
+            z=tile_index_z,
+            x=tile_index_x,
+            y=tile_index_y,
+            **kwargs
+        )
+
+
+    @distributed_trace
+    def get_copyright_caption(
+        self,
+        **kwargs: Any
+    ) -> CopyrightCaption:
+        """Copyrights API is designed to serve copyright information for Render Tile
+        service. In addition to basic copyright for the whole map, API is serving
+        specific groups of copyrights for some countries.
+
+        As an alternative to copyrights for map request, one can receive captions
+        for displaying the map provider information on the map.
+
+        :return: Get Copyright Caption Result
+        :rtype: ~azure.maps.render.models.CopyrightCaption
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        return self._render_client.get_copyright_caption(
+            **kwargs
+        )
+
+    @distributed_trace
+    def get_map_static_image(
+        self,
+        img_format: Union[str, RasterTileFormat],
+        **kwargs: Any
+    ) -> Iterator[bytes]:
+        """ The static image service renders a user-defined, rectangular image containing a map section
+        using a zoom level from 0 to 20. The static image service renders a user-defined, rectangular
+        image containing a map section using a zoom level from 0 to 20. The supported resolution range
+        for the map image is from 1x1 to 8192x8192. If you are deciding when to use the static image
+        service over the map tile service, you may want to consider how you would like to interact with
+        the rendered map. If the map contents will be relatively unchanging, a static map is a good
+        choice. If you want to support a lot of zooming, panning and changing of the map content, the
+        map tile service would be a better choice.
+
+        :param img_format:
+            Desired format of the response. Possible value: png. "png" Default value is "png".
```

Comment:
I remembered we had discussed this before, I kept it just in case there is other types will be supported in the future, and it can be added to the `RasterTileFormat` enum.